app/mixins/search_resource.js

```diff
@@ -122,10 +122,11 @@ export default Ember.Mixin.create({
    * @memberof SearchMixin
    * @returns {object} the search parameters
    **/
-  getSearchQuery() {
+  getSearchQuery(filter = undefined) {
     return {
       searchText: this.get("searchText"),
-      ...this.get("searchProps")
+      ...this.get("searchProps"),
+      filter_box_pallet: filter
```

Comment:
`search_resource` is a generic reusable mixin. It **cannot** have any references to box/pallets.

I suggest you extend the method in the component/controller which includes the mixin

Reply:
sure @patrixr , will make this change.